src/rpc/dial.rs

```diff
@@ -925,6 +950,20 @@ async fn maybe_connect_via_webrtc(
         let exchange_done = exchange_done.clone();
 
         let on_local_ice_candidate_failure = is_open_s.clone();
+
+        let caller_update_stats = caller_update_stats.clone();
+        let caller_update_stats2 = caller_update_stats.clone();
```

Comment:
I believe `caller_update_stats2` is not necessary. You should be able to clone `caller_update_stats` again because ownership of `caller_update_stats` isn't moving.

Reply:
Hmm I do think `caller_update_stats` gets moved in the `on_ice_connection_state_change` call below. If I try to reuse the value (even to clone it) in `on_ice_candidate` below _that_, the compiler complains that the value has already been moved above.

Reply:
oops! borrow semantics are confusing, disregard my above comment then :)